integration-tests/src/test/groovy/com/okta/sdk/tests/it/GroupRolesIT.groovy

```diff
@@ -0,0 +1,213 @@
+        Role userAdminRole = createdGroup1.assignRole(userAdminRoleRequest)
+
+        // 5. Add second group as the admin target
+        String href = "/api/v1/groups/" + createdGroup1.getId() + "/roles/" + userAdminRole.getId() +
+            "/targets/groups/" + createdGroup2.getId() + ""
+        userAdminRole.setResourceHref(href) //TODO: find the right way of doing this!
+
+        userAdminRole.addAdminGroup(createdGroup2.getId())
+
+        // 6. Verify the same
+        assertGroupPresent(client.listGroupTargetsForGroupRole(createdGroup1.getId(), userAdminRole.getId()), createdGroup2)
+    }
+
+    @Test
+    @Scenario("list-group-targets-for-group")
+    void removeGroupTargetFromGroupAdministratorRoleGivenToGroupTest() {
+
+        String groupName1 = "sdk-java List Test Group ${uniqueTestName}-1"
+        String groupName2 = "sdk-java List Test Group ${uniqueTestName}-2"
+        String groupName3 = "sdk-java List Test Group ${uniqueTestName}-3"
+
+        // 1. Create three groups
+        Group createdGroup1 = GroupBuilder.instance()
+            .setName(groupName1)
+            .buildAndCreate(client)
+        Group createdGroup2 = GroupBuilder.instance()
+            .setName(groupName2)
+            .buildAndCreate(client)
+        Group createdGroup3 = GroupBuilder.instance()
+            .setName(groupName3)
+            .buildAndCreate(client)
+        registerForCleanup(createdGroup1)
+        registerForCleanup(createdGroup2)
+        registerForCleanup(createdGroup3)
+
+        validateGroup(createdGroup1, groupName1)
+        validateGroup(createdGroup2, groupName2)
+        validateGroup(createdGroup3, groupName3)
+
+        // 2. List all groups and find the groups created
+        assertGroupPresent(client.listGroups(), createdGroup1)
+        assertGroupPresent(client.listGroups(), createdGroup2)
+        assertGroupPresent(client.listGroups(), createdGroup3)
+
+        // 3. Create a USER_ADMIN role request
+        AssignRoleRequest userAdminRoleRequest = client.instantiate(AssignRoleRequest)
+        userAdminRoleRequest.setType(RoleType.USER_ADMIN)
+
+        // 4. Assign the above role to the first group
+        Role userAdminRole = createdGroup1.assignRole(userAdminRoleRequest)
+
+        // 5. Add second group as the admin target for the first group
+        String href1 = "/api/v1/groups/" + createdGroup1.getId() + "/roles/" + userAdminRole.getId() +
+            "/targets/groups/" + createdGroup2.getId() + ""
+        userAdminRole.setResourceHref(href1) //TODO: find the right way of doing this!
+
+        userAdminRole.addAdminGroup(createdGroup2.getId())
+
+        // 6. Add third group as the admin target for the first group
+        String href2 = "/api/v1/groups/" + createdGroup1.getId() + "/roles/" + userAdminRole.getId() +
+            "/targets/groups/" + createdGroup3.getId() + ""
+        userAdminRole.setResourceHref(href2) //TODO: find the right way of doing this!
```

Comment:
Same comment above, don't you want to use `AddGroupTargetToGroupAdministratorRoleForGroup` here?

Reply:
Seems like it, yeah.